### ISSUE TITLE
FR: string representation

### ISSUE THREAD

**Issue by Azd325** [firebase/firebase-admin-python#349]
### [REQUIRED] Step 2: Describe your environment

  * Operating System version: Linux
  * Firebase SDK version: 3.0.0
  * Library version: 3.0.0
  * Firebase Product: cloud messages (auth, database, storage, etc)

#### Steps to reproduce:

It would be great to have a string representation of a object to have a quick preview of his contents because this is not really helpful.
```python
> str(message)
'<firebase_admin._messaging_utils.Message object at 0x11bdcda10>'

```

I think it would be great if at least the str function could present the dict variant of e.g the message Object
```python

class Message:
    def __str__(self):
        return self.__dict__

> str(Message)
{'data': None, 'notification': <firebase_admin._messaging_utils.Notification object at 0x11bdad050>, 'android': None, 'webpush': None, 'apns': None, 'fcm_options': None, 'token': None, 'topic': None, 'condition': None}


```

**Comment by hiranya911** [comment]
Sounds like a good idea. There's a custom `MessageEncoder` class that knows how to serialize all these objects. Perhaps we can use that to implement this.

**Comment by Azd325** [comment]
Would be quite nice if there some at least some human readable for the str version. :)

Thanks for the hint with `MessageEncoder`

**Comment by Azd325** [comment]
`MessageEncoder` does the job for the moment.
Is there also a way that is not a function in a protected module?

**Comment by hiranya911** [comment]
@Azd325 I wasn't suggesting you directly use `MessageEncoder`. Rather implement the functionality as a proper public API, based on `MessageEncoder`.

**Comment by lahirumaramba** [comment]
`__str__()` function is now implemented in `Message` class with the changes included in #350
